bootstrap.php

```diff
@@ -58,7 +58,7 @@ function () {
 // Performance/cache settings
 $nfd_module_container->set(
 	'cache_types',
-	array( 'browser', 'file', 'skip404' )
+	array( 'browser', 'skip404' )
```

Comment:
I'm not familiar with the reasons behind changing this, but I do see it was changed in bluehost a while ago. @wpscholar?

Reply:
We're disabling file-based caching on HostGator. My understanding is that it was never supposed to be on for HG to begin with. https://github.com/newfold-labs/wp-plugin-hostgator/pull/543/files